src/components/common/Tile.tsx

```diff
@@ -0,0 +1,191 @@
+import * as React from 'react'
+import { ActivityIndicator, TouchableWithoutFeedback, View } from 'react-native'
+import { cacheStyles } from 'react-native-patina'
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome'
+import SimpleLineIcons from 'react-native-vector-icons/SimpleLineIcons'
+
+import { Theme, ThemeProps, withTheme } from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+
+const textHeights = {
+  small: 2,
+  medium: 3,
+  large: 0
+}
+
+interface OwnProps {
+  body?: string
+  children?: React.ReactNode
+  error?: boolean
+  onPress?: () => void
+  title: string
+  type: 'editable' | 'questionable' | 'loading' | 'static' | 'touchable'
+  contentPadding?: boolean
+  maximumHeight?: 'small' | 'medium' | 'large'
+  disabled?: boolean
+}
+type Props = OwnProps & ThemeProps
+
+export class TileComponent extends React.PureComponent<Props> {
+  render() {
+    const {
+      body,
+      title,
+      contentPadding = true,
+      children,
+      theme,
+      type,
+      maximumHeight = 'medium',
+      error,
+      onPress,
+      disabled
+    } = this.props
+    const styles = getStyles(theme)
+    const numberOfLines = textHeights[maximumHeight]
+
+    if (type === 'loading') {
+      return (
+        <View>
+          <View style={styles.container}>
+            <View
+              style={[
+                styles.content,
+                contentPadding ? styles.contentPadding : null
+              ]}
+            >
+              <EdgeText style={styles.textHeader}>{title}</EdgeText>
+              <ActivityIndicator
+                style={styles.loader}
+                color={theme.primaryText}
+                size="large"
+              />
+            </View>
+          </View>
+          <View style={styles.divider} />
+        </View>
+      )
+    }
+    return (
+      <TouchableWithoutFeedback onPress={onPress} disabled={type === 'static'}>
```

Comment:
Should be 

```
<TouchableWithoutFeedback onPress={onPress} disabled={type === 'static' || disabled}>
```

Otherwise the tile will be tappable even when disabled